Widgets/openGL_widgets/AbstractGLContext.py

```diff
@@ -38,6 +40,35 @@ def handleOptionalData(self):
         except NameError:
             self.iterations = 1
 
+    def auto_center(self):
```

Comment:
I think moving this here have benefit of not repeating code, but readability is lost a little.